tests/test_examples.py

```diff
@@ -0,0 +1,49 @@
+import pathlib
+import subprocess
+
+import pytest
+from pytest_notebook import execution, notebook
+
+EXAMPLES_DIR = pathlib.Path("examples/")
+
+SH_PATHS = EXAMPLES_DIR.glob("*.sh")
+NB_PATHS = EXAMPLES_DIR.glob("*.ipynb")
+PY_PATHS = EXAMPLES_DIR.glob("*.py")
+
+
+@pytest.mark.parametrize("nb_path", NB_PATHS)
+def test_run_example_notebooks(nb_path):
+    """Smoke test ensuring that example notebooks run without error.
+
+    The `pytest_notebook` package also includes regression test functionality against
```

Comment:
FYI I have been using that regression functionality (just using `--nb-test` CLI option and default config). It works reasonably well but I have found it breaks for some annoying reasons. E.g. IIRC the notebook format changes periodically as Python versions get upgraded which will break it, until you rerun the notebook and commit to Git again. But it did catch a real breakage once too.